modules/comment-likes.php

```diff
@@ -119,7 +119,8 @@ public function frontend_init() {
 	}
 
 	public function load_styles_register_scripts() {
-		wp_enqueue_style( 'jetpack_likes', plugins_url( 'likes/style.css', __FILE__ ), array(), JETPACK__VERSION );
+		wp_enqueue_style( 'open-sans', 'https://fonts.googleapis.com/css?family=Open+Sans', array(), JETPACK__VERSION );
```

Comment:
Resolved in https://github.com/Automattic/jetpack/pull/7446/commits/068a10ab33dd93ea79676cb87e55ed2907527244